azure-pipelines.yml

```diff
@@ -106,7 +106,7 @@ jobs:
 
       # only run e2e tests when the appropriate storybook is published by scoping to relevant packages
       - script: |
-          yarn e2e $(sinceArg) --scope @fluentui/react-components
+          yarn e2e $(sinceArg) --scope @fluentui/react-components,@fluentui/babel-preset-global-context
```

Comment:
Created a new issue to track better scoping: #23455

Reply:
do we need this at all ?  I'd assume that we wanna run `e2e` task on every affected lib/app in tree that has such a task defined. can that be done or we have limited by tooling? this creates tight coupling which is "hard" to maintain from my experience.

Reply:
done in #23460